lib/Experian/IDAuth.pm

```diff
@@ -309,7 +309,15 @@ sub _get_result_proveid {
 
     return unless $credit_reference and $kyc_summary;
 
-    my $decision = {matches => []};
+    my $decision = {matches => [], kyc_summary_score => 0};
+
+    # calculate kyc summary score
+    
+    if (( my $name_count = $kyc_summary->findvalue('FullNameAndAddress/Count')) > 0
```

Comment:
These aren't going to be negative, presumably - best to leave out the `> 0` part, it makes the expression more complicated than it needs to be.

Reply:
When discussing this with francesca, she did mention that negative numbers were a possibility unfortunately.